actionpack/test/dispatch/request_id_test.rb

```diff
@@ -8,7 +8,12 @@ class RequestIdTest < ActiveSupport::TestCase
   end
 
   test "ensure that only alphanumeric uurids are accepted" do
-    assert_equal "X-Hacked-HeaderStuff", stub_request("HTTP_X_REQUEST_ID" => "; X-Hacked-Header: Stuff").request_id
+    assert_equal "X-Hacked-HeaderStuff", stub_request("HTTP_X_REQUEST_ID" => ";\r\n X-Hacked-Header: Stuff").request_id
```

Comment:
Why did this test need to change?

Reply:
It doesn't, really. I will update the PR now.